packages/grafana-prometheus/src/datasource.ts

```diff
@@ -498,6 +498,32 @@ export class PrometheusDatasource
     };
   }
 
+  async getApplicableFilters(options: DataSourceGetTagKeysOptions<PromQuery>): Promise<MetricFindValue[]> {
```

Comment:
This will not be yet implemented in prometheus, I just implemented this here to test the UI so this will not actually land.